dftimewolf/lib/collectors/grr_hosts.py

```diff
@@ -512,7 +516,9 @@ def Process(self):
     Raises:
       DFTimewolfError: if no files specified
     """
-    client = self._GetClientByHostname(self.host)
+    # TODO (tomchop): The host attribute into something more appropriate
```

Comment:
Did you mean "change the host attribute"?

Reply:
Yes, thanks. I'll push the fix and then merge the PR.